cli/running/running.go

```diff
@@ -278,7 +278,7 @@ func getInstancesFromYML(dirPath string, selectedInstName string) ([]InstanceCtx
 }
 
 // collectInstances searches all instances available in application.
-func collectInstances(appName string, appDir string) ([]InstanceCtx, error) {
+func CollectInstances(appName string, appDir string) ([]InstanceCtx, error) {
```

Comment:
Since you use this function in other modules, it makes sense to move it to some common place.

Reply:
I don't see another place for it. I can not move it to util module because of import cycles. (running->util->running) And if i want to resolve it i would need to move many other structs to util too. I don't think it is reasonable.

Reply:
> I don't see another place for it. I can not move it to util module because of import cycles. (running->util->running) And if i want to resolve it i would need to move many other structs to util too. I don't think it is reasonable.

Agree.
@LeonidVas  I think this can be done under separate refactoring task for moving all common stuff from running package to common place. We already have other commands which use running package API: `tt clean` as an example. So, these changes will affect not only the `list` command.